packages/eslint-config/package.json

```diff
@@ -27,5 +27,8 @@
     "globals": "^15.12.0",
     "typescript": "^5.7.3",
     "typescript-eslint": "^8.15.0"
+  },
+  "scripts": {
+    "clean": "rm -rf .turbo node_modules"
```

Comment:
@asutula agent spotted this, not sure if it was intentional to leave eslint-config out of the turbo clean jobs though.

Reply:
Good catch.